src/genkey/clu_genkey.c

```diff
@@ -799,6 +799,209 @@ int wolfCLU_genKey_RSA(WC_RNG* rng, char* fName, int directive, int fmt, int
 #endif
 }
 
+
+int wolfCLU_genKey_Dilithium(WC_RNG* rng, char* fName, int directive, int fmt,
+                            int keySz, int level, int withAlg, int keyType)
+{
+#ifdef HAVE_DILITHIUM
+    dilithium_key key;
```

Comment:
Lets start this out with supporting small stack. WOLFSSL_SMALL_STACK.

Reply:
Having the compile time option of putting large variables on the heap instead.

Reply:
fixed it